code/modules/cargo/delivery/package.dm

```diff
@@ -41,7 +44,7 @@
 		pay_amount = rand(12, 17) * 1000
 	if(delivery_point)
 		setup_delivery_point(delivery_point)
-	color = pick("#FFFFFF", "#EEEEEE", "#DDDDDD", "#CCCCCC", "#BBBBBB", "#FFDDDD", "#DDDDFF", "#FFFFDD", "#886600")
```

Comment:
I think this should stay
The colors here are like off-white shades of gray-ish
I think it makes sense that the courier packages aren't all perfectly shiny and white and clean
Cause they are used all over the spur and can come from anywhere

Reply:
i removed these cuz they looked a bit scuffed to me, i think it'd be better if dedicated "dirty" or "sun-bleached" sprite overlays were made, otherwise it looks like the sprite looks deep-fried imo